src/main/markdown/doc/latest/tutorial/style.md

```diff
@@ -570,27 +556,26 @@ You've already applied an HTML class attribute to the cell element to right-alig
     stocksFlexTable.getCellFormatter().addStyleName(row, 1, "watchListNumericColumn");
     stocksFlexTable.getCellFormatter().addStyleName(row, 2, "watchListNumericColumn");
     stocksFlexTable.getCellFormatter().addStyleName(row, 3, "watchListRemoveColumn");
-```
+    ```
 
 *  Instead of setting the text on the table cells, now you have to set the text for the Label widget.
 3.  Set text on the changeWidget.
     *  In the updateTable(StockPrice) method, delete the call to setText for the Change column (column 2).
     *  Create an instance of the Label widget and call it changeWidget.
     *  Set the text on changeWidget.
-    *
 
-```
-// Populate the Price and Change fields with new data.
+    ```java
+    // Populate the Price and Change fields with new data.
     stocksFlexTable.setText(row, 1, priceText);
-    <span class="strike">stocksFlexTable.setText(row, 2, changeText + " (" + changePercentText + "%)");
+    // deleted line: stocksFlexTable.setText(row, 2, changeText + " (" + changePercentText + "%)");
```

Comment:
So delete it :-) I wanted to delete those lines during the conversion to markdown but finally didn't do it because I though that could be a good information for new users. But I think it's actually more confusing